canonical-websites/www.ubuntu.com
# 1596 internet of things robotics

## Done

* update /internet-of-things/robotics to vbt
* chanded the matrix a lot… adding .is-split, .is-trisected and .is-quartered
* blockquote fixed in another PR

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/internet-of-things/robotics)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/7.%20IoT?preview=iot-robotics.png) and [demo](http://www.ubuntu.com-1596-internet-of-things-robotics.demo.haus/internet-of-things/robotics)

## Issue / Card

Fixes #1596
